blc/__main__.py

```diff
@@ -126,19 +125,31 @@ def main(args):
 
     # We build the report
     msg = 'Hello, the report of the broken link checker is ready.\n'
-    for target in broken_url:
-        msg += f"Report of {target}:\n"
-        if broken_url[target]:
-            for data in broken_url[target].items():
-                msg += str(data) + '\n'
+    for target in report:
+        msg += f"--------------\nReport of {target}\n--------------"
+        if report[target]:
+            acc = 0
+            for url, info in report[target].items():
+                if not info['result'][0]:
+                    msg += "\n"\
+                    f"URL:        {url}\n"\
+                    f"Parent URL: {info['parent']}\n"\
+                    f"Real URL:   {info['url']}\n"\
+                    f"Check time: {info['check_time']} seconds\n"\
```

Comment:
Try to reduce the floating point to 4: 
```suggestion
                    f"Check time: {round(info['check_time'], 4)} seconds\n"\
```

Reply:
Ok!